tests/drivers/can/api/src/main.c

```diff
@@ -634,6 +639,17 @@ static void test_set_bitrate_too_high(void)
 	zassert_equal(err, -ENOTSUP, "too high bitrate accepted");
 }
 
+/**
+ * @brief Test setting bitrate.
+ */
+static void test_set_bitrate(void)
+{
+	int err;
+
+	err = can_set_bitrate(can_dev, TEST_BITRATE, 0);
```

Comment:
Will this not fail for CONFIG_CAN_FD_MODE=y?

Reply:
It would, but CAN-FD is explicitly disabled for this test suite.